engine/math/vector.lisp

```diff
@@ -0,0 +1,249 @@
+(defpackage :lbge.math
+  (:use :cl))
+
+(in-package :lbge.math)
+
+(defclass float2 ()
+  ((x
+    :initform 0
```

Comment:
I'd suggest using here and anywhere with initarg+initform

```lisp
(defclass float2 ()
  ((x :initarg :x)
   (y :iniarg :y))
 (:default-initargs :x 0 :y 0))
```
to group all the initargs.

Reply:
Great idea, done